platform/linuxbsd/x11/display_server_x11.cpp

```diff
@@ -1986,12 +1986,23 @@ bool DisplayServerX11::_window_fullscreen_check(WindowID p_window) const {
 void DisplayServerX11::_validate_mode_on_map(WindowID p_window) {
 	// Check if we applied any window modes that didn't take effect while unmapped
 	const WindowData &wd = windows[p_window];
-	if (wd.fullscreen && !_window_fullscreen_check(p_window)) {
-		_set_wm_fullscreen(p_window, true, wd.exclusive_fullscreen);
-	} else if (wd.maximized && !_window_maximize_check(p_window, "_NET_WM_STATE")) {
-		_set_wm_maximized(p_window, true);
-	} else if (wd.minimized && !_window_minimize_check(p_window)) {
-		_set_wm_minimized(p_window, true);
+
+	bool desiredFullscreen = _window_fullscreen_check(p_window);
```

Comment:
We use snake_case for identifiers, so this should be `desired_fullscreen`. Same for others.

*Note:* Please make modifications by _amending_ the commit, and *force pushing* the changes, so that it stays as a single commit (see [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#modifying-a-pull-request)).

Reply:
My bad, the old habit from C#. Now fixed.